File: week-02/day-01/arrays/src/SumAll/SumAll.java
package SumAll;

import java.util.Arrays;

public class SumAll {
  public static void main(String[] args) {
    int[] numbers = {3, 4, 5, 6, 7};
    int sum = 0;

    for (int i = 0; i < numbers.length; i++) {
      sum += numbers[i];
    }
    System.out.println("Sum of elements: " + sum);
  }
}

File: week-11/arrays/sum-all.js
// -  Create a variable named `numbers` with the following content:
//   `[3, 4, 5, 6, 7]`
// -  Log the sum of the elements in `numbers` to the console

let numbers = [3, 4, 5, 6, 7];
let sum = 0;

numbers.forEach(element => {
    sum = sum + element;
});

console.log(sum);